src/origin_sdk/OriginSession.py

```diff
@@ -153,6 +153,43 @@ def __inputs_gql_request_with_loading_handler(
 
         return data
 
+    def __inputs_gql_workbook_status(self, scenario_id: str):
+        request_id = None
+
+        while True:
+            variables = {
+                "sessionId": scenario_id,
+                "requestId": request_id,
```

Comment:
What's the difference between a `request_id` of `None` and an actual `request_id`?